src/components/dropdown-menu/dropdown-menu.stories.js

```diff
@@ -0,0 +1,123 @@
+import DropdownMenu from './dropdown-menu.jsx';
+import Avatar from '../avatar/avatar.jsx';
+import Menu from '../menu-item/menu-item.jsx';
+import Button from '../button/button.jsx';
+import { House } from 'lucide-react';
+
+DropdownMenu.displayName = 'DropdownMenu';
+DropdownMenu.Trigger.displayName = 'DropdownMenu.Trigger';
+DropdownMenu.Content.displayName = 'DropdownMenu.Content';
+
+Menu.displayName = 'Menu';
+Menu.List.displayName = 'Menu.List';
+Menu.Item.displayName = 'Menu.Item';
```

Comment:
Instead of setting the display name inside the story can we put it from the component file?